BasicSample/Assets/Shared/Scripts/HideHandSpheresWhenNotActive.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR;
+
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    public class HideHandSpheresWhenNotActive : MonoBehaviour
```

Comment:
nit: "HideHandSpheresWhenNotActive" -> "HideHandSpheresWhenNotTracked" could make the purpose more clear from the script title alone

Reply:
you bet!

Reply:
changed